tools/make-pkgs/make-build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# If insufficent args provided, exit
+if [ -z "$1" ] || [ -z "$2" ]; then
```

Comment:
as you're already using bash, follow bashisms to avoid pitfalls...
like `[[ ... || ... ]]` instead of `[ ... ] || [ ... ]`

Reply:
I hardcoded /bin/bash because /bin/sh might point to other shells like zsh/fish/nushell etc,.
Otherwise I tried to keep the scripts as POSIX-compliant as possible.
This was intentional.

Reply:
if you're thinking about posix compatibility, you have to use `/bin/sh` because `/bin/sh` is the de facto posix compliant shell... all shell will behave the same if invoked as /bin/sh (it doesn't matter whether it is linked to bash/zsh/dash). [^1]

> I hardcoded /bin/bash because /bin/sh might point to other shells like zsh/fish/nushell etc

And, no sane person will link `/bin/sh` to posix incompatible shell (namely fish/nushell/...) unless that person is Derek Taylor (DT) who's brain-dead anyway...[^2][^3]

You can be sure that /bin/sh will never be linked to a posix incompatible shell, if you're trying to be posix-sh compliant pedantically, just use `/bin/sh`.

[^1]: https://en.wikipedia.org/wiki/Bourne_shell
[^2]: https://www.youtube.com/watch?v=xQ4KJF-6Agc [Read the comments]
[^3]: https://www.youtube.com/watch?v=8AQobpqySAU

Reply:
After putting some thought into it, I have decided to use the [[ ]] bash double brackets for all usecases (except one where [ ] makes sense)